cmd/podman/build.go

```diff
@@ -155,6 +155,10 @@ func buildCmd(c *cliconfig.BuildValues) error {
 		}
 	}
 
+	if _, err := os.Stat(c.BudResults.Authfile); err != nil {
```

Comment:
I think the problem with the test is you first need to check if the Authfile has been set.  This gets a little wonky due to the Cobra includes, but I think you can check that with `c.Flag("authfile").Changed`.   You'll need to play/test that making sure that gets tripped appropriately as you pass the auth-file option into the build command.

Reply:
this `c.Flag("authfile").Changed`only works for those check that directory runs os.stats under /cmd.
maybe I should get rid of shared.GetSystemContext, just `c.Flag("authfile").Changed`and os.stats for each cmd